pkg/sql/logictest/testdata/logic_test/owner

```diff
@@ -255,3 +255,121 @@ user testuser
 
 statement error pq: role testuser cannot be dropped because some objects depend on it\nowner of database d\nowner of schema d.public\nowner of schema d.s\nowner of table d.public.t\nowner of table d.s.t\nowner of table test.public.t\nowner of type d.public._typ\nowner of type d.public.typ
 DROP ROLE testuser
+
+user root
+
+skipif config local-mixed-24.3
```

Comment:
nit: could you instead declare a subtest that gets skipped in the mixed version config, to avoid all of these `skipif config local-mixed-24.3` lines?

Reply:
you can't.

Reply:
yeah, perhaps it's best not to mess with logic.go